### PR TITLE
Task/task 3

### PR DESCRIPTION
**Pull request by Krush21git**
Add a new API Project that will allow you to consume the sale class and add a new sale

### CODE REVIEW

File: IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs [hunk 1]
```diff
@@ -19,15 +19,21 @@ public IndustryConnectWeek2Context(DbContextOptions<IndustryConnectWeek2Context>
 
     public virtual DbSet<CustomerSale> CustomerSales { get; set; }
 
+    public virtual DbSet<CustomerSaleView> CustomerSaleViews { get; set; }
+
+    public virtual DbSet<CustomerSaleView1> CustomerSaleView1s { get; set; }
```
Comment (kestermvp) [inline IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs:24]: What's their difference?

File: IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs [hunk 2]
```diff
@@ -19,15 +19,21 @@ public IndustryConnectWeek2Context(DbContextOptions<IndustryConnectWeek2Context>
...
+
     public virtual DbSet<Product> Products { get; set; }
 
     public virtual DbSet<Sale> Sales { get; set; }
 
     public virtual DbSet<Store> Stores { get; set; }
 
+    public virtual DbSet<Store1> Store1s { get; set; }
```
Comment (kestermvp) [inline IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs:32]: Why do you need to set up another Store in here?

File: IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs (hunk 1 above)
Comment (Krush21git) [reply IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs:24]: Are you asking about the CustomSaleViews and CustomerSaleView1 ?

It is created for some extra practice for queries in week2.

File: IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs (hunk 2 above)
Comment (Krush21git) [reply IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs:32]: It is created for some extra practice for queries in week2.
but now i don't need that table so i will remove it.

**Pull request merged**